### PR TITLE
bazel: add instrumentation_filter to collect all coverage

### LINKED ISSUE #48868: Coverage of `pkg` is not collected when running integration tests
## Enhancement

For example, `make bazel_addindextest3` should have covered `pkg/ddl/backfilling_operators.go`.

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #48868

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
   ```
   $ head -n 10 tests/realtikvtest/addindextest3/addindextest3_test/coverage.dat 
   mode: set
   github.com/pingcap/tidb/pkg/statistics/index.go:45.33,46.16 1 1
   github.com/pingcap/tidb/pkg/statistics/index.go:49.2,55.25 3 1
   github.com/pingcap/tidb/pkg/statistics/index.go:58.2,58.21 1 1
   github.com/pingcap/tidb/pkg/statistics/index.go:61.2,61.25 1 1
   github.com/pingcap/tidb/pkg/statistics/index.go:64.2,64.21 1 1
   github.com/pingcap/tidb/pkg/statistics/index.go:67.2,69.11 3 1
   github.com/pingcap/tidb/pkg/statistics/index.go:46.16,48.3 1 0
   github.com/pingcap/tidb/pkg/statistics/index.go:55.25,57.3 1 0
   github.com/pingcap/tidb/pkg/statistics/index.go:58.21,60.3 1 0
   ```
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

**Review by ywqzzy — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -497,7 +497,7 @@ bazel_test: failpoint-enable bazel_prepare
 
 
 bazel_coverage_test: failpoint-enable bazel_ci_simple_prepare
-	bazel $(BAZEL_GLOBAL_CONFIG) --nohome_rc coverage $(BAZEL_CMD_CONFIG) --jobs=35 --build_tests_only --test_keep_going=false \
+	bazel $(BAZEL_GLOBAL_CONFIG) --nohome_rc coverage $(BAZEL_CMD_CONFIG) --jobs=35 --instrumentation_filter=/pkg --build_tests_only --test_keep_going=false \
```
Comment (wuhuizuo) [inline Makefile:500]: It does not contain `br` and `dumpling` paths.

Comment (hawkingrei) [reply Makefile]: Done

**Comment by hawkingrei** [comment]
/unhold

remove UT's ```instrumentation_filter```

**Pull request merged**